demo/src/screens/componentScreens/SectionsWheelPickerScreen.tsx

```diff
@@ -0,0 +1,108 @@
+import _ from 'lodash';
+import React, {useState, useCallback} from 'react';
+import {Alert} from 'react-native';
+import {Text, View, SectionsWheelPicker, SegmentedControl, Button} from 'react-native-ui-lib';
+import {WheelPickerProps} from '../../../../src/incubator/WheelPicker';
```

Comment:
We're not exporting Incubator WheelPickerProps in our src/index?

Reply:
No, adding it